extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -188,6 +189,12 @@ describe('amp-a4a', () => {
     const re = /^([^;]+);(\d+);([\s\S]*)$/;
     const match = re.exec(name);
     expect(match).to.be.ok;
+    const sandbox = child.getAttribute('sandbox');
```

Comment:
Don't use the variable name 'sandbox', use something like fastFetchSandbox or renderingSandbox, sandbox is already used as part of the testing framework. Fix here and below.

Reply:
Renamed to "sandboxAttribute" since it's an attribute on the iframe.